lib/modules/hosts/default.js

```diff
@@ -2,12 +2,8 @@ export default {
 	moduleID: 'default',
 	name: 'default',
 	domains: [],
-	detect({ href }) {
-		const acceptRegex = /^[^#]+?\.(gif|jpe?g|png)(?:[?&#_].*|$)/i;
-		const rejectRegex = /(?:wikipedia\.org\/wiki|(?:i\.|m\.)?imgur\.com|photobucket\.com|gifsound\.com|gfycat\.com|\/wiki\/File:.*|reddit\.com|onedrive\.live\.com|500px\.(?:com|net|org)|(?:www\.|share\.)?gifyoutube\.com)|futurism\.co/i;
-
-		return acceptRegex.test(href) && !rejectRegex.test(href);
-	},
+	regex: /^[^#]+?\.(gif|jpe?g|png)(?:[?&#_].*|$)/i,
+	detect({ href }) { this.regex.test(href); },
```

Comment:
Just use an arrow function with the regex inline, like we have in most hosts. V8 (and probably IonMonkey) cache compiled regexes, so it's not expensive like you might expect.

Or allow `detect` to be a regex or a function.